inst/qml/mlPrediction.qml

```diff
@@ -82,7 +82,7 @@ Form
 			CheckBox
 			{
 				name:							"addPredictors"
```

Comment:
addFeatures

Reply:
There's no compatibility issue with an upgrader implemented by Joris. The use of the upgrader might now be easier. @Kucharssim did you recently used and upgrader?

Reply:
Sure, upgrades work fine! Docs here: https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-upgrade-qml.md